src/SprykerEco/Zed/Unzer/Business/Credentials/UnzerCredentialsValidator/UnzerCredentialsStoreRelationsValidator.php

```diff
@@ -85,7 +85,7 @@ public function validate(UnzerCredentialsTransfer $unzerCredentialsTransfer): Un
      */
     protected function isUnzerCredentialsHaveStoreRelations(UnzerCredentialsTransfer $unzerCredentialsTransfer): bool
     {
-        if (in_array($unzerCredentialsTransfer->getTypeOrFail(), UnzerConstants::UNZER_CHILD_CONFIG_TYPES, true)) {
+        if (in_array((int)$unzerCredentialsTransfer->getTypeOrFail(), UnzerConstants::UNZER_CHILD_CONFIG_TYPES, true)) {
```

Comment:
Imho we don't need cast here because getTypeOrFail already returns int or not?

Reply:
We need to cast here cause Symfony form in another module builds object in such way so here could be string